pyproject.toml

```diff
@@ -12,5 +12,5 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<3.8"
+python = ">=3.8,<3.9"
```

Comment:
I was surprised you didn't just go to `>=3.7,<3.9` but I think this is fine if you don't anticipate we'll need to do more testing in 3.7.